runner/runner.go

```diff
@@ -49,29 +49,22 @@ func (c Command) String() string {
 	return b.String()
 }
 
-func NewCommand(argv []string, env map[string]string, timeout time.Duration, snapshotId string) *Command {
-	return &Command{Argv: argv, EnvVars: env, Timeout: timeout, SnapshotId: snapshotId}
+func NewCommand(argv []string, env map[string]string, timeout time.Duration) *Command {
+	return &Command{Argv: argv, EnvVars: env, Timeout: timeout}
 }
 
-type Runner interface {
-	// Run instructs the Runner to run cmd. It returns its status and any errors.
-	// Run may:
-	// enqueue cmd (leading to state PENDING)
-	// run cmd immediately (leading to state RUNNING)
-	// check if cmd is well-formed, and reject it if not (leading to state FAILED)
-	// wait a very short period of time for cmd to finish
-	// Run may not wait indefinitely for cmd to finish. This is an async API.
-	Run(cmd *Command) (ProcessStatus, error)
-
-	// Status checks the status of run.
-	Status(run RunId) (ProcessStatus, error)
-
-	// Current status of all runs, running and finished, excepting any Erase()'s runs.
-	StatusAll() ([]ProcessStatus, error)
+// Service allows starting/abort'ing runs and checking on their status.
+type Service interface {
```

Comment:
Agree. Also, am not a big fan of the alias, unless we are really planning on cleaning it up soon, this is the type of thing that is easy to deprioritize forever and increase cruft.